.gitignore

```diff
@@ -50,3 +50,5 @@ cypress/videos
 
 # Cypress screenshots
 cypress/screenshots
+
+.DS_Store
```

Comment:
I'm still seeing a few `.DS_Store` files have been committed below:

![chrome_40gJZ3IfeX](https://user-images.githubusercontent.com/13315440/149506979-b24828ff-0ae0-4019-83d7-9c3ef3831ec8.png)

Worth removing them manually, and possibly using the following to also match subdirectories
```suggestion
**/.DS_Store
```

Reply:
Actually I think just `.DS_Store` should work, but it is tracking them anyway because they have already been committed.

Reply:
This file should be gone now, as of the last commit

Reply:
I can see the `.DS_Store` was committed on the third commit (SHA 582d34f) so you might need to rebase or do an interactive rebase to remove it from there.

Reply:
I'm still seeing 3 .DS_Store files on here.